package.json

```diff
@@ -98,6 +102,8 @@
     "jest-extended": "^2.0.0",
     "jest-transform-stub": "^2.0.0",
     "metro-react-native-babel-preset": "^0.66.2",
+    "patch-package": "^6.5.0",
+    "postinstall-postinstall": "^2.1.0",
```

Comment:
Is this postinstall-postinstall package really necessary. This is only needed if we need to execute a postinstall after removing a package. This is not the case, is it?